flow/CMakeLists.txt

```diff
@@ -191,3 +194,6 @@ if(APPLE)
   target_link_libraries(flow PRIVATE ${IO_KIT} ${CORE_FOUNDATION})
   target_link_libraries(flow_sampling PRIVATE ${IO_KIT} ${CORE_FOUNDATION})
 endif()
+
+add_executable(mkcert MkCertCli.cpp)
+target_link_libraries(mkcert PUBLIC fmt::fmt flow)
```

Comment:
It would be good to add some ctests calling mkcert, otherwise we commit untested code

Reply:
The next PR will be having local_cluster.py and tmp_cluster.py run mkcert for TLS-enabled testing. Since mkcert is strictly test-only (as stated in its --help output), I find it odd to have to write test for them. It would be like refusing to merge tmp_cluster.py unless there's a test to make sure that a cluster is actually spun up.

That said, I hand-tested the CLI with various inputs to confirm. Plus there's a CLI option to actually print out cert chains in human-readable form.

Reply:
It is just a suggestion. I won't block the merge